src/main/java/duke/register/Parser.java

```diff
@@ -0,0 +1,71 @@
+package duke.register;
+/**
+ * This class processes the command by breaking down into components
+ */
+public class Parser {
+    private String command;
+    private String[] commandSeparate;
+    private String taskType;
+    private String taskName;
+    private String date = "";
+    private String index;
+
+    public Parser(String command) {
+        this.command = command;
+        commandSeparate = command.split(" ");
```

Comment:
Maybe commandsSplittedByWhiteSpace or just commandsSplitted might be a clearer name?